src/farmers/farmerbot/farmerbot_quick.md

```diff
@@ -40,76 +68,184 @@ Here are some examples to guide you:
+    [Install]
+    WantedBy=multi-user.target     
+    ```
+* Enable the Farmerbot service
+    ```
+    systemctl daemon-reload
+    systemctl enable farmerbot
+    systemctl start farmerbot
+    ```
+* Verify that the Farmerbot service is properly running
+    ```
+    systemctl status farmerbot
+    ```
 
-* Create the parent folder
-  * ```
-    mkdir -p farmerbot_docker && cd $_
+## Method 2: Docker
+
+We present how to run the Farmerbot using Docker.
+
+### Build the Farmerbot Docker Image
+
+- [Install Docker on the VM](../../computer_it_basics/docker_basics.md#install-docker-desktop-and-docker-engine)
     ```
-    * Note: you can choose a different name for your parent folder
-* Pull the file creator image from the GitHub Container Registry
-  * ```
-    docker pull ghcr.io/threefoldtech/farmerbot_config:0.2.0
+    curl -fsSL https://get.docker.com -o get-docker.sh
+    sh get-docker.sh
     ```
-* Run the file creator image and answer the questions to generate the files
-  * ```
-    docker run --name fbot_config_container -v ./:/farmerbot -ti ghcr.io/threefoldtech/farmerbot_config:0.2.0
+- Clone the the ThreeFold Tech `tfgrid-sdk-go` repository and switch to the latest release tag
+    ```
+    git clone https://github.com/threefoldtech/tfgrid-sdk-go
+    git checkout tags/v0.13.16
+    ```
+- Build the Farmerbot Docker image
+    ```
+    cd tfgrid-sdk-go/farmerbot  
+    docker build -t farmerbot -f Dockerfile ../
     ```
-    * The `config/config.md` and `.env` files are saved in the parent folder
 
-> Note: It is can be a good idea to set only one node with `yes` for `never_shutdown`. A node with low power consumption is obviously a wise choice.
+### Create the Farmerbot Files
 
-***
+- Create the Farmerbot directory
+    ```
+    cd ..
+    mkdir farmerbotgo
+    cd farmerbotgo
+    ```
+- Create the Farmerbot `config.yml` file (see template below)
+    ```
+    nano config.yml
+    ```
+- Create the environment variables file and set the variables (see template below)
+    ```
+    nano .env
+    ```
 
-## Run the Farmerbot
+### Run the Farmerbot
 
-* Download the Farmerbot `.yaml` file with this command:
-  * ```
-    wget https://raw.githubusercontent.com/threefoldtech/farmerbot/development/docker-compose.yaml
+- Run the Farmerbot docker image in the background
     ```
-* Run the Farmerbot with this command
-  * ```
-    docker compose up -d
+    docker run -d --restart unless-stopped -v $(pwd)/config.yml:/config.yml -v $(pwd)/.env:/.env farmerbot run -e /.env -c /config.yml
     ```
 
-***
+## Farmerbot Files
+
+### Configuration File Template (config.yml)
+
+In this example, the farm ID is 1, we are setting the Farmerbot with 4 nodes and the node 1 never shuts down, we set a periodic wakeup at 1:00PM.
+
+Note that the timezone of the farmerbot will be the same as the time zone of the machine the farmerbot running inside. By default, a full VM on the TFGrid will be set in UTC.
+
+```
+farm_id: 1
+included_nodes:
```

Comment:
please add a note if the user omitted included nodes, it'll be adding all of the nodes in the farm

Reply:
Done. I also added a quick url link to the additional information section, which covers all those nuances.